src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -223,6 +223,11 @@ public void TestTypeExpressions()
             VerifySyntax<TypeSyntax>(_g.ArrayTypeExpression(_g.ArrayTypeExpression(_g.IdentifierName("x"))), "x[][]");
             VerifySyntax<TypeSyntax>(_g.NullableTypeExpression(_g.IdentifierName("x")), "x?");
             VerifySyntax<TypeSyntax>(_g.NullableTypeExpression(_g.NullableTypeExpression(_g.IdentifierName("x"))), "x?");
+
+            VerifySyntax<TypeSyntax>(_g.TupleTypeExpression(_g.TupleElementExpression(_g.IdentifierName("x"), "y") ), "(x y)");
```

Comment:
Using SyntaxNode instead of ITypeSymbol it is possible to use the SyntaxGenerator without using Compilation and adding references. I added overloads that accept ITypeSymbol instead of SyntaxNode since that seems customary. When creating tuples with less than 2 elements the methods throw.